bake/bake.go

```diff
@@ -558,7 +558,7 @@ func (c Config) newOverrides(v []string) (map[string]map[string]Override, error)
 			// IMPORTANT: if you add more fields here, do not forget to update
 			// docs/bake-reference.md and https://docs.docker.com/build/bake/overrides/
```

Comment:
not related but comment is not quite correct:

```suggestion
			// docs/reference/buildx_bake.md (--set) and https://docs.docker.com/build/bake/overrides/
```

Reply:
Ah I was looking for where to change this.